### PR TITLE
Update ceos node with new pattern

### PR DESCRIPTION
**Pull request by shubh90**

### CODE REVIEW

File: topo/node/ceos/ceos.go [hunk 1]
```diff
@@ -263,38 +269,54 @@ func defaults(pb *tpb.Node) *tpb.Node {
...
-		Labels: map[string]string{
+		}
+	}
+	if pb.Labels == nil {
+		pb.Labels = map[string]string{
 			"type":    tpb.Node_ARISTA_CEOS.String(),
 			"vendor":  tpb.Vendor_ARISTA.String(),
 			"model":   pb.Model,
 			"os":      pb.Os,
 			"version": pb.Version,
```
Comment (alexmasi) [inline topo/node/ceos/ceos.go:280]: Should the map values here and in "Env" only get merged if the initial field is nil or should these values be added to the map regardless of the value?

Comment (marcushines) [reply topo/node/ceos/ceos.go:280]: they should be merged

Comment (marcushines) [reply topo/node/ceos/ceos.go:280]: if they aren't "present" overwrite them

Comment (shubh90) [reply topo/node/ceos/ceos.go:282]: Done. Made "Label" and "Env" merge and overwrite with defaults if not present

**Review by marcushines — APPROVED** [review]

**Pull request merged**